fuse/dl/README.md

```diff
@@ -0,0 +1,75 @@
+1. `LossSegmentationCrossEntropy` - Cross-entropy loss with optional class weights and resizing of the ground truth map.
+2. `BinaryDiceLoss` - Dice loss with optional class weights, `batch_dict` preprocessing and ground truth map resizing. 
+3. `FocalLoss` - Focal loss with configurable hyperparameters and optional scalar weight, `batch_dict` preprocessing and ground truth map resizing.  
+
+Note again, that while these are specific loss functions that are implemented, it is possible to use any loss, whether custom, or one that already exists in PyTorch, and pass it to the `LossDefault` class for use in FuseMedML.
+
+## models
+This module contains DL model and architecture related classes. 
+The most basic class is `ModelWrapSeqToDict`. It is FuseMedML's wrapper for PyTorch models to be used in fuse. It is initialized with an existing PyTorch module, and a list of input and output keys. When `forward` is called on a fuse `batch_dict`, it extracts the data from the input keys from `batch_dict`, calls the model `forward` function on the data, and writes the output to the `batch_dict`'s output keys.  
+Optionally, a pre and post processing function can be provided to be applied on the `batch_dict` before and after model run.  
+[The MNIST example uses `ModelWrapSeqToDict`](../../examples/fuse_examples/imaging/classification/mnist/run_mnist.py)
+
+ Additional basic classes include:
+ 1. `ModelEnsemble` - Initialized from a list of trained models directory, this class can run several models sequentially. It then produces a dictionary with predictions of each model in the ensemble, as well as average and majority vote over the predictions.
+ 2. `ModelMultiHead` - A class which given backbone and multiple heads, implements a neural network with the corresponding structure.  
+ [The STOIC 21 example uses `ModelMultiHead`](../../examples/fuse_examples/imaging/classification/stoic21/runner_stoic21.py)  
+ 3. `ModelMultistream` - Implements a neural network with multiple backbone processing streams and heads.
+ 4. `ModelSiamese` - Implements a siamese neural network, with two identical backbones and multiple heads.
+
+Besides these basic classes, more specific architecture blocks are implemented. They are divided into "backbones" and "heads".
+
+Implemented backbones include a "vanilla" fully connected network, or Multi Layer Perceptron (MLP), supported versions of 2D and 3D ResNets, and an Inception ResNet.  
+[The KNIGHT challenge example uses a 3D ResNet backbone `BackboneResnet3D`](../../examples/fuse_examples/imaging/classification/knight/baseline/fuse_baseline.py)  
+
+Implemented "heads" include a number of parameterized classifier heads, in 1D, 2D and 3D, as well as a dense segmentation head.  
+[The KNIGHT challenge example uses a 3D classification head `Head3DClassifier`](../../examples/fuse_examples/imaging/classification/knight/baseline/fuse_baseline.py)
+
+## optimizers
```

Comment:
Can you delete it and also the code.
It currently not supported (we didn't port it to fuse2)